src/igvfd/schemas/changelogs/image.md

```diff
@@ -2,6 +2,7 @@
 
 ### Minor changes since schema version 4
 
+* Remove `thumb_nail`.
```

Comment:
Can we say "Remove calculated property thumb_nail"? I think this will be more consistent with how we have recorded the addition of new calculated properties e.g. "Add calculated property controlled_access."